External/FEXCore/unittests/Emitter/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+if (ENABLE_VIXL_DISASSEMBLER)
+  option(CATCH_BUILD_STATIC_LIBRARY "" ON)
```

Comment:
This is already done in the main CMakeLists.txt and shouldn't be needed anywhere else (with the exception of FEXLinuxTests, which is cross-compiled as an external project).

Reply:
Resolved.